scipy/stats/stats.py

```diff
@@ -3999,6 +3999,52 @@ def pearsonr(x, y):
     >>> stats.pearsonr([1, 2, 3, 4, 5], [10, 9, 2.5, 6, 4])
     (-0.7426106572325057, 0.1505558088534455)
 
+    It is important to keep in mind that no correlation does not imply
+    independence unless (X, Y) is Gaussian. Correlation can even be zero
```

Comment:
All of these appear in the documentation:
```
x and y
X and Y
`x` and `y`  # (This PR would remove these from the mstats version, but the theme still stylizes parameters like this)
:math:`x` and :math:`y`
``x`` and ``y``
```
Consider defining the symbols used here or reusing symbols from above to emphasize the distinction, which may be important when we are referring to e.g. a distribution vs a sample.

Reply:
it is indeed confusing. I tried to simplify the notation and just used x, y and r. It should be clear from the context if it refers to a sample / parameter / distribution. let me know what you think. I felt that using double backticks would lead to strange expression sometimes, e.g. to describe the bivariate distribution (x, y). happy to change it again...